src/spritekit.cs

```diff
@@ -256,20 +260,36 @@ partial interface SKNode : NSCoding, NSCopying {
 		[Export ("moveToParent:")]
 		void MoveToParent (SKNode parent);
 
+		[Availability (Deprecated = Platform.iOS_10_0 | Platform.Mac_10_12)]
```

Comment:
it looks like it was *removed* in beta 1 headers, which is not the same as deprecation. It turns out they were *moved*

    iOS-Beta1/GameplayKit-iOS-Beta1.md:++ (NSArray<GKPolygonObstacle*> *)obstaclesFromNodePhysicsBodies:(NSArray<SKNode*>*)nodes;
    iOS-Beta1/SpriteKit-iOS-Beta1.md:-+ (NSArray<GKPolygonObstacle*> *)obstaclesFromNodePhysicsBodies:(NSArray<SKNode*>*)nodes;

which is *fine* because of the way they are declared (category). It's still worth to comment them with the change.